index.html

```diff
@@ -1068,10 +1065,10 @@ <h4>
       handler and one used specifically for a single write.
     </p>
     <pre class="example">
-      const reader = new NDEFReader();
+      const ndef = new NDEFReader();
       let ignoreRead = false;
 
-      reader.onreading(event => {
+      ndef.onreading(event => {
```

Comment:
This is not valid in JS. Please do not forget to remove extra `)` at the end of this function.

```suggestion
      ndef.onreading = (event) => {

Reply:
fixed, thanks

Reply:
It is still not fixed.